docs: add changelogs (#233)
qase-tms/qase-python: qase-pytest/changelog.md

# qase-pytest 6.1.0b1

## What's new

> Implement a method which constructs the signature to each test result.
> It takes the file path, the suites, the qase ids and the parameters.

Reviewer: ```suggestion
Implemented a method that constructs a signature for each test result.
It takes the file path, suites, qase IDs, and parameters.
```

Author: fixed

# qase-pytest 6.0.3

## What's new

> Fixed the issue then specified suite in the decorator was not displayed. Fix [#231]

Reviewer: ```suggestion
Fixed the issue where the suite specified in the decorator was not displayed. Fix [#231]
```

Author: fixed

# qase-pytest 6.0.2

## What's new

Fixed the issue [#226]:

```log
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```

# qase-pytest 6.0.1

## What's new

> Change a name of the complete test run parameter for CLI arguments. Fix [#160]

Reviewer: ```suggestion
Changed the name of the complete test run parameter for CLI arguments. Fix [#160]
```

Reviewer: what's the old and the new name?

Author: old name - qase_testops_run_description
new name - qase_testops_run_complete

but it was in the internal logic. For clients, the names have not changed.

# qase-pytest 6.0.0

## What's new

> This is the first release in the 6.0.x series of the Pytest reporter.

Reviewer: ```suggestion
The first release in the 6.0.x series of the Pytest reporter.
```

Author: fixed

# qase-pytest 6.0.0b2

## What's new

> The sleep profiler was turned off until the test data collection was completed.

Reviewer: ```suggestion
Turn off the sleep profiler until the test data collection is completed.
```

Author: fixed
